lnwallet/test/test_interface.go

```diff
@@ -1213,6 +1213,23 @@ func testListTransactionDetails(miner *rpctest.Harness,
 			spew.Sdump(txDetails))
 	}
 
+	// Fetch two transactions with the txHashList parameter.
+	txHashes := []*chainhash.Hash{&txDetails[0].Hash, &txDetails[1].Hash}
+	txDetails, err = alice.ListTransactionDetails(0, 0, txHashes, "")
+	for i := range txHashes {
+		require.NoError(t, err, "unexpected error")
+		require.NotEqual(t, txDetails, 1, "expected one transaction, "+
+			"got: %v", len(txDetails))
+		require.NotEqual(t, txDetails[i].NumConfirmations,
+			numBlocksMined, "expected %v confirmations, got: %v",
+			numBlocksMined, txDetails[i].NumConfirmations)
+		require.NotEqual(t, txDetails[i].Value, outputAmt, "tx value "+
+			"incorrect, got %v expected %v", txDetails[i].Value,
+			outputAmt)
+		require.NotEqual(t, txDetails[i].BlockHash[:], blocks[i][:],
```

Comment:
Why `NotEqual` everywhere here? We're not really asserting anything with those statements?

Reply:
Sorry for committing this test code. It is horrible in every way.